### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by frafishen**
update readme

### CODE REVIEW

File: docs/0-Building-env-file.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+- `POSTGRES_PORT`: The port on which PostgreSQL listens.
+
+Example:
+```env
+POSTGRES_USER="riceball"
+POSTGRES_PASSWORD="gp12345"
+POSTGRES_DB="postgres"
+POSTGRES_HOST="postgres_db"
+POSTGRES_PORT="5432"
+```
```
Comment (yun-si) [inline docs/0-Building-env-file.md:24]: I think the passward should not be placed directly?

Comment (yun-si) [reply docs/0-Building-env-file.md:24]: all the other look good to me~~

File: docs/0-Building-env-file.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+- `POSTGRES_USER`: The username for the PostgreSQL database.
+- `POSTGRES_PASSWORD`: The password for the PostgreSQL database.
+- `POSTGRES_DB`: The default database to connect to.
+- `POSTGRES_HOST`: The hostname of the PostgreSQL container.
+- `POSTGRES_PORT`: The port on which PostgreSQL listens.
+
+Example:
+```env
+POSTGRES_USER="riceball"
+POSTGRES_PASSWORD="gp12345"
```
Comment (yun-si) [inline docs/0-Building-env-file.md:20]: I think the password should not place here directly.
Or it's ok?

all the other looks good to me~

Comment (frafishen) [reply docs/0-Building-env-file.md]: i've removed this part!

**Review by yun-si — APPROVED** [review]

**Pull request merged**